### ISSUE TITLE
Screen Reader: Landing Page: Social media links have no accessible names

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74853]
## Action Performed
1. Using JAWS + Chrome, open the page https://new.expensify.com/
2. Navigate using TAB key to the social media links in the footer, located under 'Learn More'
3. Observe the announcements

## Expected Result
Each of the social media links located in the footer will announce with a unique name and role. For example:

-"Twitter, link"
-"Instagram, link"
-"Facebook, link"
-"Linkedin, link"

## Actual Result
The social media links, located at the bottom footer have no accessible name. When focused, they all announce as the same thing:

"Link graphic, Expensify"

<a href="[https://www.twitter.com/expensify"](https://www.twitter.com/expensify%22) data-tag="pressable" tabindex="-1" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1d4mawv r-14gqq1x" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="24" height="24" hovered="false" pressed="false" fill="#E7ECE9"><path d="M16.2 6.2c-.5.2-1 .4-1.5.4.5-.4.9-.9 1.1-1.4-.5.2-1 .5-1.6.6-.3-.4-1.1-.8-1.8-.8-1.6 0-2.9 1.5-2.5 3.1-2.1 0-4-1-5.2-2.6-.7 1.1-.5 2.6.8 3.5-.5-.1-.9-.2-1.3-.4 0 1.2.9 2.2 2 2.5-.2.1-.6.1-1.1.1.4 1 1.2 1.8 2.4 1.8-1.1.9-2.4 1.2-3.8 1 1.1.8 2.5 1.1 3.9 1.1 4.8 0 7.5-4 7.2-7.6.7-.3 1.1-.7 1.4-1.3"></path></svg></div></a>

## Area issue was found
Landing Page

## Failed Checkpoint
WCAG 2.4.4 Link Purpose (In Context)
https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context.html

## User Impact
People with motion impairment can skip links that they are not interested in, avoiding the keystrokes needed to visit the referenced content and then returning to the current content. People with cognitive limitations will not become disoriented by multiple means of navigation to and from content they are not interested in. People with visual disabilities will be able to determine the purpose of a link by exploring the link's context.

## Suggested Resolution
Provide an accessible name to each of the social media links using aria-label

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/5ae6fe2a-43c1-4608-abeb-9a9aa905dd74

<img width="1917" height="999" alt="Image" src="https://github.com/user-attachments/assets/bf029d30-e4d1-4f07-a18f-59e0605547e1" />

**Version Number:**NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995179
**Issue reported by:** NA
**Slack conversation:** NA
